kml/serdes_gpx.cpp

```diff
@@ -173,6 +181,12 @@ void GpxParser::Pop(std::string_view const & tag)
       }
       else if (GEOMETRY_TYPE_LINE == m_geometryType)
       {
+        TrackLayer layer;
+        layer.m_lineWidth = kDefaultTrackWidth;
+        layer.m_color.m_predefinedColor = PredefinedColor::None;
```

Comment:
What is this color?

Reply:
Yes, was not able to find it's usage, removed.